wordpress-mobile/WordPress-iOS
# Update CocoaPods to the 1.7 series for Xcode 11

It's all in the title 😃

**To Test:**
- Run `rm -rf Pods && bundle install && bundle exec pod install` on this branch, then build and run the app.
- Ensure all tests continue to pass.
